lib/tasks/create_persona_users.rake

```diff
@@ -0,0 +1,5 @@
+desc 'Creates a number of persona provider user accounts'
+task create_persona_users: :environment do
```

Comment:
could you update the docs to include this rake task, as it no longer lives in the local dev setup?